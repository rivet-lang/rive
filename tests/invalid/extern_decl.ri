extern (C) {
    pub fn abort() void; // OK
}

mod module {
    extern pkg oreo; // FAIL
    extern (C) {
        fn abort() void; // OK
    }
}

extern (C) fn sss() { } // OK
