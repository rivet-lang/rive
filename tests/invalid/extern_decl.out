tests/invalid/extern_decl.ri:6:16: error: package `oreo` not found
rivetc: error: could not compile package `extern_decl`, aborting due to previous error